config/jobs/kubernetes/sig-k8s-infra/oci-proxy/sig-k8s-infra-oci-proxy-presubmits.yaml

```diff
@@ -0,0 +1,46 @@
+presubmits:
+  kubernetes-sigs/oci-proxy:
+  - name: pull-oci-proxy-verify
+    decorate: true
+    always_run: true
+    optional: true
+    spec:
+      containers:
+      - image: golang
+        command:
+        - make
+        - verify
+        # shellcheck has its own job so we don't need docker
+        - VERIFY_SHELLCHECK=false
+  - name: pull-oci-proxy-shellcheck
+    decorate: true
+    always_run: true
+    optional: true
+    spec:
+      containers:
+      # NOTE: We will have to keep this up to date with the repo ...
```

Comment:
TODO: we can maybe loosen the version check in the repo to a warning to make this easier.
I think it's worth not needing DinD